tools/code_format/check_format.py

```diff
@@ -860,12 +862,15 @@ def checkSourcePath(file_path):
       error_messages += executeCommand(command, "header_order.py check failed", file_path)
     command = ("%s %s | diff %s -" % (CLANG_FORMAT_PATH, file_path, file_path))
     error_messages += executeCommand(command, "clang-format check failed", file_path)
-
+  
   if file_path.endswith(PROTO_SUFFIX) and isApiFile(file_path):
     package_name, error_message = packageNameForProto(file_path)
     if package_name is None:
       error_messages += error_message
-  return error_messages
+
+  if file_path.endswith(PROTO_SUFFIX):
```

Comment:
Why not use the above `if` block? We only want to do this to API files.

Reply:
sure, will do it.